pyannote/metrics/spotting.py

```diff
@@ -63,15 +74,20 @@ class LowLatencySpeakerSpotting(BaseMetric):
     """
 
     @classmethod
-    def metric_name(cls):
+    def metric_name(cls) -> str:
         return "Low-latency speaker spotting"
 
-    @classmethod
-    def metric_components(cls):
-        return {'target': 0.}
+    def metric_components(self) -> MetricComponents:
```

Comment:
Not quite sure about this change.

Components are meant to be accumulated, which is not the case for those "components" you added.

Reply:
Not all. Only those related to components.